README.adoc

```diff
@@ -12,15 +12,13 @@ For samples, documentation, and usage information, please see http://ehcache.org
 
 == Current release
 
-We released the Milestone 5 on February 20th, 2016.
-This milestone is feature complete for Ehcache 3 and marks the start of release candidates production where the focus
-will be on performance and documentation.
+We released 3.0.0 on April 15th 2016.
 
-The https://github.com/ehcache/ehcache3/releases/tag/v3.0.0.m5[release notes] contain the links to the documentation to help you get started
+The https://github.com/ehcache/ehcache3/releases/tag/v3.0.0[release notes] contain the links to the artifacts and the documentation to help you get started.
 
 == Current development & next release
 
-We are now working on producing release candidates.
-Expect mostly performance and documentation improvements in addition to the usual bug fixes.
+We are now working on the clustering tier of Ehcache 3 which will be released as 3.1.0 later.
```

Comment:
This might be better expressed as "the clustering tier of Ehcache 3 which will be later released as 3.1.0."